### DISCUSSION TITLE
Auth helper not working properly

### DISCUSSION THREAD

**Discussion by aldairandre** [clerk/javascript#2807] [Support request - DO NOT OPEN HERE]
**Nextjs 14.0.4**
**Clerk ^4.29.7**
This is my first time using clerk, according to the clerk documentation on [auth() page](https://clerk.com/docs/references/nextjs/auth#auth) , the auth() function should return the Authentication object, but for some reason that I don't know, some of these properties are returning undefined, properties with user , session, protect and among others.

**Page code that is a server component**
![auth-code-test](https://github.com/clerk/javascript/assets/72257475/2bf3424a-820c-405d-8f36-dfee8f49116f)

**Terminal exit**
![auth-console](https://github.com/clerk/javascript/assets/72257475/1480955d-9990-49de-8ecd-2bbd3f249bd1)

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
